### PR TITLE
[IMP] multi-website snippets with the same key

### PR DESCRIPTION
**Pull request by yelizariev**
**Description of the issue/feature this PR addresses:**
One more step for multi-website support. 

**Current behavior before PR:**
Several snippets with the same key but for different websites raises error

**Desired behavior after PR is merged:**
Allow duplicate snippets to have different versions for different websites

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web_editor/models/ir_qweb.py [hunk 1]
```diff
@@ -46,7 +46,11 @@ class QWeb(models.AbstractModel):
 
     def _compile_directive_snippet(self, el, options):
         el.set('t-call', el.attrib.pop('t-snippet'))
-        name = self.env['ir.ui.view'].search([('key', '=', el.attrib.get('t-call'))]).display_name
+        name = self.env['ir.ui.view'].search([
+            ('key', '=', el.attrib.get('t-call')), '|',
```
[the remark this answers is not in the document]
Comment (yelizariev) [reply addons/web_editor/models/ir_qweb.py]: good point

**Comment by Yenthe666** [comment]
@JKE-be for you to check when you have time.

**Comment by yelizariev** [comment]
Updated and squased

```

-        ], order='website_id DESC')[:1].display_name
+        ], order='website_id DESC', limit=1).display_name

```

**Comment by JKE-be** [comment]
Landed in master https://github.com/odoo/odoo/commit/4f9d9d169f65369e02dfde122150364a2d646c5c

Thanks for PR

**Pull request closed without merging**